lib/repository_cache.rb

```diff
@@ -0,0 +1,26 @@
+# Interface to the Redis-backed cache store used by the Repository model
+class RepositoryCache
+  attr_reader :namespace
+
+  def initialize(namespace, backend = Rails.cache)
```

Comment:
Why the second argument? [Dependency Injection](http://en.wikipedia.org/wiki/Dependency_injection)